package.json

```diff
@@ -11,7 +11,6 @@
   "dependencies": {},
   "devDependencies": {
     "@financial-times/n-express": "^17.20.2",
```

Comment:
Is this an opportunity to try and update to n-express 19 too? I think it's a pretty big change